zio-http/src/main/scala/zio/http/Form.scala

```diff
@@ -0,0 +1,71 @@
+package zio.http
+
+import java.net.URLEncoder
+import java.nio.charset.Charset
+import java.net.URLDecoder
+import zio._
+
+/**
+ * Represents a form that can be either multipart or url encoded.
+ */
+sealed trait Form { self =>
```

Comment:
Perhaps `FormData` instead of `Form`?

Reply:
That was my first preference, but there is already a `FormData` elsewhere in the codebase :(

Reply:
Maybe we can consolidate?

Reply:
I am using now using `FormData`.  However, I don't see a reasonable way to combine it as the other variant is only related to `ContentDisposition`.